nucliadb_sdk/nucliadb_sdk/v2/sdk.py

```diff
@@ -364,6 +365,32 @@ def _check_response(self, response: httpx.Response):
         response_type=ResourceFieldAdded,
     )
 
+    # Configuration
+    set_configuration = _request_builder(
+        name="set_configuration",
```

Comment:
nit `create_configuration` (like other `POST` calls)

Reply:
This function does not create , just sets the config so the name is better like labels.